pkg/kubecfg/vm.go

```diff
@@ -83,6 +84,18 @@ const (
 	RegistryResolver
 )
 
+// Enum value maps for ResolverType.
+var (
+	ResolverType_name = map[ResolverType]string{
+		0: "noop",
+		1: "registry",
+	}
+	ResolverType_value = map[string]ResolverType{
+		"noop":     0,
+		"registry": 1,
+	}
+)
+
```

Comment:
Let's use specific enum values for easy the reading:

```suggestion
// Enum value maps for ResolverType.
var (
	ResolverType_name = map[ResolverType]string{
		NoopResolver: "noop",
		RegistryResolver: "registry",
	}
	ResolverType_value = map[string]ResolverType{
		"noop":     NoopResolver,
		"registry": RegistryResolver,
	}
)

```